test/parallel/test-cluster-disconnect-exitedAfterDisconnect-race.js

```diff
@@ -19,7 +20,7 @@ let eventFired = false;
 cluster.worker.disconnect();
 
 process.nextTick(common.mustCall(() => {
-  assert.strictEqual(eventFired, false, 'disconnect event should wait for ack');
+  assert.ok(eventFired, 'disconnect event should wait for ack');
```

Comment:
Isn't this changing the check from `eventFired === false` to, basically, `!!eventFired`?

Reply:
Whoops, didn't commit all my changes...let's try again...